TTCN3_Project/CANFilter/com/testingtech/playits/canfilter/TimeoutResponder.java

```diff
@@ -0,0 +1,67 @@
+package com.testingtech.playits.canfilter;
+
+import java.net.Socket;
+import java.util.Hashtable;
+import java.util.Timer;
+
+public class TimeoutResponder {
+
```

Comment:
Is it really necessary to have the following three variables declares as class variables? They are only used in addtimer().